scripts/needs_testrun.py

```diff
@@ -237,13 +230,13 @@ def pr_matches_patterns(patterns: t.Set[str]) -> bool:
     return bool([_ for p in patterns for _ in fnmatch.filter(changed_files, p)])
 
 
-def extract_git_commit_selections(git_commit_message: str) -> list:
+def extract_git_commit_selections(git_commit_message: str) -> t.Set[str]:
```

Comment:
Did we have duplicates before?

Reply:
no it was a set already that was returned as a sorted list